src/GitHubExtension/DeveloperId/LoginUIController.cs

```diff
@@ -25,378 +38,288 @@ public ProviderOperationResult Initialize(IExtensionAdaptiveCard extensionUI)
     {
         Log.Logger()?.ReportDebug($"Initialize");
         _loginUI = extensionUI;
-        var operationResult = _loginUI.Update(_loginUITemplate.GetLoginUITemplate(LoginUIState.LoginPage), null, LoginUIState.LoginPage);
-        return operationResult;
+        return new LoginPage().UpdateExtensionAdaptiveCard(_loginUI);
     }
 
     public IAsyncOperation<ProviderOperationResult> OnAction(string action, string inputs)
     {
         return Task.Run(async () =>
         {
+            if (_loginUI == null)
+            {
+                Log.Logger()?.ReportError($"_loginUI is null");
+                return new ProviderOperationResult(ProviderOperationStatus.Failure, null, "_loginUI is null", "_loginUI is null");
+            }
+
             ProviderOperationResult operationResult;
-            Log.Logger()?.ReportInfo($"OnAction() called with state:{_loginUI?.State}");
+            Log.Logger()?.ReportInfo($"OnAction() called with state:{_loginUI.State}");
             Log.Logger()?.ReportDebug($"action: {action}");
 
-            switch (_loginUI?.State)
+            switch (_loginUI.State)
             {
-                case LoginUIState.LoginPage:
-                {
-                    try
+                case nameof(LoginUIState.LoginPage):
                     {
-                        // If there is already a developer id, we should block another login.
-                        if (DeveloperIdProvider.GetInstance().GetLoggedInDeveloperIdsInternal().Any())
+                        try
                         {
-                            Log.Logger()?.ReportInfo($"DeveloperId {DeveloperIdProvider.GetInstance().GetLoggedInDeveloperIdsInternal().First().LoginId} already exists. Blocking login.");
-                            _loginUI.Update(_loginUITemplate.GetLoginUITemplate(LoginUIState.LoginFailedPage), null, LoginUIState.LoginFailedPage);
-                            operationResult = new ProviderOperationResult(ProviderOperationStatus.Failure, null, "Only one DeveloperId can be logged in at a time", "One DeveloperId already exists");
-                            break;
-                        }
+                            // If there is already a developer id, we should block another login.
+                            /*if (_developerIdProvider.GetLoggedInDeveloperIdsInternal().Any())

```

Comment:
Remove commented out code.

Reply:
This shouldn't be commented out. This essentially blocks Multi-user login since Widgets don't support multi-user yet. I just removed it for the sake of testing GHES. Will fix before completing this PR.